src/raft/uv_metadata.c

```diff
@@ -182,7 +180,7 @@ int uvMetadataStore(struct uv *uv, const struct uvMetadata *metadata)
 	unsigned short n;
 	int rv;
 
-	assert(metadata->version > 0);
+	PRE(1 <= metadata->format_version && metadata->format_version < 3);
```

Comment:
I think it is better if we use the constants here instead of 1 and 3 :P

Reply:
Yep!